responses/accounts/public-scope.json

```diff
@@ -0,0 +1,11 @@
+{
+  "accounts": [
+    {
+      "business_name": "Business Name",
+      "created_at": "CREATED_AT",
```

Comment:
@Fredar Can you please advise on the ordering logic here. Please also file a ticket or make a PR with a note about this on the ReadMe > Contributing.